pegasus/sites.v3/code.org/styles_min/040-page.css

```diff
@@ -294,6 +294,14 @@ input[type="submit"] {
       align-items: center;
       gap: 0.75rem;
     }
+    &.has-external-link {
```

Comment:
Nit: add whitespace line above this line.

Reply:
Updated!